portfolio/src/main/webapp/style.css

```diff
@@ -129,4 +129,10 @@ table, tr, td  {
     flex: 100%;
     max-width: 100%;
   }
+}
+
+.label{
+    background-color: #FF9900;
+    color:#cccccc;
+    font-size: 16px;
 }
```

Comment:
New line

Reply:
can you clarify what you meant by "new line"